src/ai/backend/client/cli/admin/sessions.py

```diff
@@ -32,7 +51,9 @@
 @click.option('-a', '--all', is_flag=True,
               help='Display all sessions matching the condition using pagination.')
 @click.option('--detail', is_flag=True, help='Show more details using more columns.')
-def sessions(status, access_key, id_only, show_tid, dead, running, all, detail):
+@click.option('-f', '--format', default=None,  help='Display only specified fields.')
+@click.option('--plain', is_flag=True, help='Display process status in plain format.')
```

Comment:
"process status" looks somewhat ambiguous.
Let's say "Display the session list without decorative line drawings and the header."

Reply:
Also don't forget to update the same option in `ps.py`.

Reply:
Got it!